Scripts/Player.cs

```diff
@@ -173,4 +219,19 @@ public override void _Process(float delta)
 
         onFloor = this.IsOnFloor();
     }
+
+    public float GetCurrentAir()
```

Comment:
Can we use properties for these to keep consistency with the rest of the code. e.g. public float CurrentAir { get; set; } or something